### PR TITLE
Yaseen

### PR DESCRIPTION
**Pull request by Yassotaha**
/What do you think of my form 2 ?

### CODE REVIEW

File: Projet-VBNet/Form2.resx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿<root>
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" />
+    <xsd:element name="root" msdata:IsDataSet="true">
+      <xsd:complexType>
+        <xsd:choice maxOccurs="unbounded">
+          <xsd:element name="metadata">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" />
```
Comment (Yassotaha) [inline Projet-VBNet/Form2.resx:10]: here you can find a generted code

**Comment by Yassotaha** [comment]
can u merge this please ?

**Review by Alar31000 — APPROVED** [review]
Yes ! Go Live

**Review by InesFotso — COMMENTED** [review]
Bien reçu

**Pull request merged**